tests/lwip_sock_ip/Makefile

```
@@ -24,6 +24,7 @@ USEMODULE += netdev_eth
 USEMODULE += netdev_test
 USEMODULE += ps
 USEMODULE += sock_ip
+USEMODULE += sock_aux_local
```

... (cont'd)

Maybe make this switchable like IPv4/IPv6?

```suggestion
AUX_LOCAL ?= 1
ifneq (0, $(AUX_LOCAL))
  USEMODULE += sock_aux_local
endif
```

(but please move it up than to the other switches and not just apply the suggestion blindly, please ;-).

---

Done. (Also moving it up)